docs/release-process.md

```diff
@@ -142,11 +142,10 @@ This only needs to be done once.
     ```
 
 8. Announce the release on Matrix and the mailing list.
-   TODO: Move announcement templates into this repository.
-   Release managers can copy and paste the previous release's announcement for
-   now.
-   Make sure to change the version numbers and sha256sum in the announcement
-   text.
+
+    ```
+    antsibull-build announcements --send --data-dir ${MAJOR_VERSION} ${VERSION}
+    ```
```

Comment:
I would make this match the entry in the automated docs. GitHub won't let me make a suggestion on the unchanged line 144, though.

Reply:
Adjusted in 5ad0ad71749cb2907b8cdd6115246bbd5d9b8045. I also adjusted this document's title similarly to the automated one.